airflow/www/fab_security/sqla/manager.py

```diff
@@ -243,6 +244,9 @@ def add_role(self, name: str) -> Optional[Role]:
                 self.get_session.commit()
                 log.info(c.LOGMSG_INF_SEC_ADD_ROLE.format(name))
                 return role
+            except IntegrityError as e:
+                log.warning(c.LOGMSG_ERR_SEC_ADD_ROLE.format(str(e)))
+                self.get_session.rollback()
```

Comment:
We are already catching the `Exception` on L250 which should also take care of catching `IntegrityError`